### ISSUE TITLE
Form Expiration

### ISSUE THREAD

**Issue by CarlSteven** [DataAnalyticsinStudentHands/FormBuilder#25]
Forms should have an expiration date beyond which they become "disabled"

Beyond this date, it should be treated as a disabled form as requested in #5

**Comment by CarlSteven** [comment]
We discussed this today as having a service-level check for having passed expiration date, and if so changing the enabled=false in the service level.

**Comment by CarlSteven** [comment]
Running into issue where  the expiration datetime field keeps “truncating” the time portion of the field. It only saves the 2/12/2012 portion, not the 1:10PM part of it.

Going off of Chrome's network tab:
I send it -> expiration_date: "2015-02-05T22:00:00.000Z"
It gives me -> expiration_date: 1423094400000

MySQL:
It is only storing 2015-02-05 in the field in the table.

I'm going to hide this stuff in the settings page so it's not confusing.

**Comment by cmholley** [comment]
The mySQL database was updated, with the expiration_date field changed from DATE to DATETYPE. An issue still exists with replacing dates with NULLS to remove expiration dates.

**Comment by cmholley** [comment]
Back end issues fixed, waiting on front end changes

**Comment by CarlSteven** [comment]
Implemented front-end, need help on clearing this value, if user no longer wants a expiration date.

**Comment by Tswiggs** [comment]
Oops opened a separate issue #103.  This is now fixed for both the form itself and for formResponses once edit functionality is available.
